docs/docsite/links.yml

```diff
@@ -1,45 +1,17 @@
 ---
-# This will make sure that plugin and module documentation gets Edit on GitHub links
-# that allow users to directly create a PR for this plugin or module in GitHub's UI.
-# Remove this section if the collection repository is not on GitHub, or if you do not want this
-# functionality for your collection.
 edit_on_github:
-  repository: ansible-collections/community.REPO_NAME
+  repository: kubevirt/kubevirt.core
   branch: main
-  # If your collection root (the directory containing galaxy.yml) does not coincide with your
-  # repository's root, you have to specify the path to the collection root here. For example,
-  # if the collection root is in a subdirectory ansible_collections/community/REPO_NAME
-  # in your repository, you have to set path_prefix to 'ansible_collections/community/REPO_NAME'.
-  path_prefix: ''
-
-# Here you can add arbitrary extra links. Please keep the number of links down to a
-# minimum! Also please keep the description short, since this will be the text put on
-# a button.
-#
-# Also note that some links are automatically added from information in galaxy.yml.
-# The following are automatically added:
-#   1. A link to the issue tracker (if `issues` is specified);
-#   2. A link to the homepage (if `homepage` is specified and does not equal the
-#      `documentation` or `repository` link);
-#   3. A link to the collection's repository (if `repository` is specified).
 
 extra_links:
   - description: Report an issue
-    url: https://github.com/ansible-collections/community.REPO_NAME/issues/new/choose
+    url: https://github.com/kubevirt/kubevirt.core/issues/new/choose
+  - description: Repository with sources
+    url: https://github.com/kubevirt/kubevirt.core
```

Comment:
This link isn't strictly necessary, since that link is already automatically added (from https://github.com/kubevirt/kubevirt.core/blob/main/galaxy.yml#L20) with title "Repository (Sources)".

Reply:
Dropped.